package.json

```diff
@@ -12,9 +12,10 @@
     "@apidevtools/swagger-parser": "10.0.3",
     "@autorest/schemas": "^1.3.4",
     "@azure-tools/openapi-tools-common": "^1.2.2",
-    "@azure/core-http": "^3.0.1",
-    "@azure/core-util": "~1.7.0",
-    "@azure/logger": "~1.0.4",
+    "@azure/core-http": "3.0.4",
```

Comment:
Pin packages from azure-sdk to ensure compat with TS 3.9.  Newer versions require newer TS, which includes breaking changes.